packages/contracts-bedrock/contracts/test/OptimismPortal.t.sol

```diff
@@ -1129,7 +1129,7 @@ contract OptimismPortalResourceFuzz_Test is Portal_Initializer {
         // Bound resource config
         _maxResourceLimit = uint32(bound(_maxResourceLimit, 21000, MAX_GAS_LIMIT / 8));
         _gasLimit = uint64(bound(_gasLimit, 21000, _maxResourceLimit));
-        _prevBaseFee = uint128(bound(_prevBaseFee, 0, 5 gwei));
+        _prevBaseFee = uint128(bound(_prevBaseFee, 0, 3 gwei));
```

Comment:
With this value above 3, certain cases result in a situation where more than MAX_GAS_LIMIT (30MM) is required to cover the requested L2 gas limit.